### ISSUE TITLE
SentinelAutoConfiguration#ObjectMaper 冲突问题

### ISSUE THREAD

**Issue by xiejiashuai** [spring-cloud-incubator/spring-cloud-alibaba#150]
- 在类中有这么一段代码
```java
  ...
       @Bean("sentinel-json-converter")
	public JsonConverter jsonConverter(
			@Qualifier("sentinel-object-mapper") ObjectMapper objectMapper) {
		return new JsonConverter(objectMapper);
	}

	@Bean("sentinel-object-mapper")
	public ObjectMapper objectMapper() {
		return new ObjectMapper();
	}
   ...
```

用户自身应用内部可能也会声明一个`ObjectMapper `，最终容器内部会有两个`ObjectMapper `实例，最终导致容器失败的原因是在Boot中的如下代码
```java
@Configuration
class JacksonHttpMessageConvertersConfiguration {

	@Configuration
	@ConditionalOnClass(ObjectMapper.class)
	@ConditionalOnBean(ObjectMapper.class)
	@ConditionalOnProperty(name = HttpMessageConvertersAutoConfiguration.PREFERRED_MAPPER_PROPERTY, havingValue = "jackson", matchIfMissing = true)
	protected static class MappingJackson2HttpMessageConverterConfiguration {

		@Bean
		@ConditionalOnMissingBean(value = MappingJackson2HttpMessageConverter.class, ignoredType = {
				"org.springframework.hateoas.mvc.TypeConstrainedMappingJackson2HttpMessageConverter",
				"org.springframework.data.rest.webmvc.alps.AlpsJsonHttpMessageConverter" })
		public MappingJackson2HttpMessageConverter mappingJackson2HttpMessageConverter(
				ObjectMapper objectMapper) {
			return new MappingJackson2HttpMessageConverter(objectMapper);
		}

	}
}
```

能否把Sentinel这个`ObjectMapper `bean 声明去掉呢？

**Comment by fangjian0423** [comment]
Yes, we will remove `@Bean` annotation in `SentinelAutoConfiguration#objectMapper` and `SentinelAutoConfiguration.SentinelXmlConfiguration#xmlMapper` because it is no necessary to create a bean in `ApplicationContext`.

**Comment by excepts** [comment]
Why you guys only implemented the jsonCoverter ObjectMapper of Jackson, if I configure to exclude all jackson JSON tools, instead of which I used fastjson. No corresbponding this jsonCoverter.
Well, although I can do as below:
`@Bean("sentinel-json-flow-converter")
    Converter jsonConverter() {
        return (Converter<String, List<AbstractRule>>) source ->
                StringUtils.isBlank(source) ? Lists.newArrayList(): JSON.parseArray(source, AbstractRule.class);
    }` 
But only for this case, I found annother one in dubbo integration stuff.

Come on, man , dubbo, alibaba's stuff, why only object mapper only , but not friendly for his brother fastjson.    I would log an issue later
